### DISCUSSION TITLE
Unreal SDK Update 2.7.0

### DISCUSSION THREAD

**Discussion by YVRDevelopers** [PlayForDreamDevelopers/support#20] [Announcements]
**新增
YvrXR**
- 新增**空间锚点**，用于定位和跟踪物体或场景的关键点
- 新增**房间标定**，开发者可获取用户创建的房间标定数据，并将其使用在应用中。
- 合成层（**Compositor Layer**）支持**Cubemap**形状
- 新增**动态分辨率（Preview）**，可以根据GPU负载自动调节渲染分辨率

**下载地址**：https://developer.yvr.cn/yvrdoc/sdk/unreal/UnrealSDK_2_7_0.zip

**Added
YvrXR**
- Added **spatial anchors** for locating and tracking key points of objects or scenes.
- Added **scene SDK**, developers can obtain space setup data created by users and use it in applications.
- **Compositor Layer** supports **Cubemap** shape.
- Added **Adaptive resolution**, allows your app to automatically adjust the screen resolution based on the app's GPU workload.

**Download Link**: https://developer.yvr.cn/yvrdoc/sdk/unreal/UnrealSDK_2_7_0.zip

**Comment by danieldownes** [comment]
1: Is there a timeline to support eye-tracking in the Unreal plugin?
2: Does it work in OpenXR, or will it be in the Unreal SDK only?
3: How about when wirelessly tethered?
